### DISCUSSION TITLE
Does ". Lotus / datastore / chain" support customization?

### DISCUSSION THREAD

**Discussion by ChaiWenyu1983** [filecoin-project/lotus#9197] [Build and Sync] [answered]
The ". Lotus / datastore / chain" file grows too fast. Can you customize it to other directories?

**Comment by rjan90** [comment, marked as answer]
Hey @ChaiWenyu1983!

You can customize the directory where your `.lotus`-folder is located by exporting the enviroment variable `LOTUS_PATH=/path/to/folder`. It defaults to `~/.lotus` if it´s not set.

If you only want to customize the path of the `.lotus/datastore/chain` folder, I recommend symlinking it 😄
